### PR TITLE
Add user story issue template

### PR DESCRIPTION
**Pull request by signekb**
Based on the discussions yesterday and today, I have created an issue template based on the user story template.

This is very simple/minimalistic, which might be nice, actually? 

My plan would be to describe this template in the Git guidelines - with examples of [user], [functionality], and [benefit].

### CODE REVIEW

**Comment by lwjohnst86** [comment]
Good starting point! Shared repo infrastructure should be placed in the `team` repo so that we can synch with all other repos there. Though for this, it would be under the `.github` repo (we haven't created it) (details here: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file) I'll make an issue for making this in the `team` repo. For now, we can merge this in here and figure out the details for the other things later.

**Comment by signekb** [comment]
Ah, I see. So merge for now and create a default-community-health-file soon?

**Comment by lwjohnst86** [comment]
@signekb You got it!!

**Comment by signekb** [comment]
@lwjohnst86 Sooo, are you going to approve? :-P

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: User story
+about: Suggest a feature from the perspective of a specific user role
```
Comment (lwjohnst86) [inline .github/ISSUE_TEMPLATE/user-story.md:3]:
```suggestion
about: Suggest a feature from the perspective of a specific user
```

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+As a [user], 
+
+I want to [functionality], 
+
+so that [benefit].
+
+
+**Relates to issue:** #
```
Comment (lwjohnst86) [inline .github/ISSUE_TEMPLATE/user-story.md:17]: Do we need this or better to convert the other issue into this format? Or do you mean connecting this template to related issues to keep track of?

File: .github/ISSUE_TEMPLATE/user-story.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+labels: ''
+assignees: ''
+
+---
+
+As a [user], 
+
+I want to [functionality], 
+
+so that [benefit].
```
Comment (lwjohnst86) [inline .github/ISSUE_TEMPLATE/user-story.md:14]: Any reason we should have these on separate lines instead of one?

**Review by lwjohnst86 — COMMENTED** [review]
Oooo, I just found out about [Issue Forms](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms)! That could be very useful!

I just made some small comments.

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 1 above)
Comment (signekb) [reply .github/ISSUE_TEMPLATE/user-story.md:3]: Maybe "specific type of user" instead? "Specific user" makes it sound like one specific user?

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 3 above)
Comment (signekb) [reply .github/ISSUE_TEMPLATE/user-story.md:14]: Not really. That was just how it was set up when they showed us at the workshop. One line instead is fine by me.

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 2 above)
Comment (signekb) [reply .github/ISSUE_TEMPLATE/user-story.md:17]: I am actually not sure why I put that in there. I believe I took it from GitHub's feature request template.

**Comment by signekb** [comment]
I'll start working on a(n initial) .github repo 👍

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 2 above)
Comment (lwjohnst86) [reply .github/ISSUE_TEMPLATE/user-story.md:17]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/user-story.md (hunk 3 above)
Comment (lwjohnst86) [reply .github/ISSUE_TEMPLATE/user-story.md:14]:
```suggestion
As a [user], I want to [functionality], so that [benefit].
```

**Pull request merged**